vm/devices/storage/disk_nvme/nvme_driver/fuzz/fuzz_emulated_device.rs

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//! A shim layer to fuzz responses from an emulated device.
+use crate::arbitrary_bool;
+use crate::get_raw_data;
+
+use arbitrary::Unstructured;
+use chipset_device::mmio::MmioIntercept;
+use chipset_device::pci::PciConfigSpace;
+use inspect::Inspect;
+use inspect::InspectMut;
+use pci_core::msi::MsiInterruptSet;
+use user_driver::DeviceBacking;
+use user_driver::emulated::{EmulatedDevice, Mapping, EmulatedDmaAllocator, DeviceSharedMemory};
```

Comment:
Nit, keep imports separated. There's a rust-analyzer setting for this that we mention in the onboarding guide.

Reply:
Fixed